release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -627,6 +627,10 @@ The bundle format is the preferred Operator packaging format for Operator Lifecy
 
 For more information, see xref:../operators/operator_sdk/osdk-pkgman-to-bundle.adoc#osdk-pkgman-to-bundle[Migrating package manifest projects to bundle format].
 
+[id="ocp-4-8-service-ca-operator-enhancement"]
+==== Service-ca Operator enhancement
+{product-title} 4.8 allows users to run `service-ca-operator` pods as a non-root user to suit their organization's needs. When ran as a non-root user, the `service-ca-operator runs with an user ID of `UID=1001(1001) GID =1001 groups=1001`. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1914446[*BZ#1914446*]. 
```

Comment:
Missing final ` for service-ca-operator.

Because I had to point that out, I'd also use "When run..." to match the verb tense (?) from the first instance of "to run."

Reply:
+1 to _When run_.